.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```

Comment:
Please add to `.gitignore` the entire directory under `.idea`. That's IDE specific config that other people don't use.

Reply:
Done :)